### PR TITLE
user best practices draft

### PR DESCRIPTION
**Pull request by mrjones-plip**

### CODE REVIEW

File: user-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### New contact & new user, associate with existing place
+
+This is top most recommended practice.  The steps are:
+
+1. Have the old CHW do one last sync to ensure all data is pushed up from their device
+1. Create a new contact for the new CHW.
+1. Make this contact the new primary contact for the old CHW's place - do not create a new place!
+1. Change the name the CHW's place as needed
+1. Create a new user  associated to the new contact.
+1. Change the password for the old CHW's user.
```
Comment (garethbowen) [inline user-best-practices.md:20]: The two steps about managing the old user (the last one, and the first one) are optional and sort of separate. You can have two CHWs in the same area with the same contacts. I would separate those two steps out under a subheading, for example...

```suggestion
1. Create a new contact for the new CHW.
1. Make this contact the new primary contact for the old CHW's place - do not create a new place!
1. Change the name the CHW's place as needed
1. Create a new user  associated to the new contact.

If you want to deactivate the old CHW

1. Have the old CHW do one last sync to ensure all data is pushed up from their device
1. Change the password for the old CHW's user on the server
1. Wipe the user's device to clear any private health information
```

File: user-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+This is straightforward and a single step: just change the name on the 3 records.  This practice is not recommended, but offers a lot of real world benefits.
+
+#### Pros
+* Very few steps to set up
+* No extra CHW user and contact left to linger
+
+#### Cons
+* Humans and computers alike looking at the CHT, CouchDB and Posgtress can **NOT** tell both who was giving the care, and when the switch happened between caregivers.
+* If the old CHW returns and you want to keep both CHWs active, you're stuck creating a new contact and user.
+* Insecure - the old CHW can still log in on another device
```
Comment (garethbowen) [inline user-best-practices.md:45]: I'd recommend changing the password at the same time so at least the old CHW can't log in again.

File: user-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+This is by far the simplest practice - effectively no steps are needed in the CHT.  It comes with many draw backs and is not recommended. The pros and cons are similar to prior option.
+
+#### Pros
+* No setup
+* No extra CHW user and contact left to linger
+
+#### Cons
+* Humans and computers alike looking at the CHT, CouchDB and Posgtress can **NOT** tell both who was giving the care, and when the switch happened between caregivers.
+* If the old CHW returns and you want to keep both CHWs active, you're stuck creating a new contact and user AND getting a new device.
+* Insecure - the old CHW can still log in on another device
```
Comment (garethbowen) [inline user-best-practices.md:58]: Again, you can still change the password while handing the device over.

**Review by garethbowen — COMMENTED** [review]
Great work laying out the options!

**Comment by mrjones-plip** [comment]
Thanks @garethbowen

**Comment by mrjones-plip** [comment]
closing as this was just a place to review the content before posting on [the forum](https://forum.communityhealthtoolkit.org/t/best-practices-for-user-management-at-scale/1668/2?u=mrjones).

**Pull request closed without merging**